contrib/inventory/foreman.py

```diff
@@ -198,6 +198,25 @@ def _get_json(self, url, ignore_errors=None, params=None):
                 break
         return results
 
+    def _use_inventory_report(self):
```

Comment:
Interesting approach. I was thinking about catching a 404 on the `schedule_report`.